bolt/utils.py

```diff
@@ -127,3 +128,38 @@ def slicify(slc, dim):
     else:
         raise ValueError("Type for slice %s not recongized" % type(slc))
 
+def prime_factors(n):
```

Comment:
I'd hold off adding this for now, as it's not used in the codepath introduced in this PR. We'll hopefully bring it back soon!